src/SprykerEco/Zed/Unzer/Business/Credentials/UnzerCredentialsCreator.php

```diff
@@ -177,4 +192,25 @@ protected function createMainMerchantUnzerCredentials(
             $childUnzerCredentialsResponseTransfer->getUnzerCredentialsOrFail(),
         );
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\UnzerCredentialsTransfer $unzerCredentialsTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerCredentialsTransfer
+     */
+    protected function expandUnzerCredentialsWithParentStoreRelations(UnzerCredentialsTransfer $unzerCredentialsTransfer): UnzerCredentialsTransfer
+    {
+        $parentIdUnzerCredentials = $unzerCredentialsTransfer->getParentIdUnzerCredentialsOrFail();
+
+        $unzerCredentialsCriteriaTransfer = (new UnzerCredentialsCriteriaTransfer())
+            ->setUnzerCredentialsConditions(
+                (new UnzerCredentialsConditionsTransfer())->addId($parentIdUnzerCredentials),
+            );
+        $parentUnzerCredentialsTransfer = $this->unzerReader->findUnzerCredentialsByCriteria($unzerCredentialsCriteriaTransfer);
+        if ($parentUnzerCredentialsTransfer === null) {
```

Comment:
you can invert if condition and put `$unzerCredentialsTransfer->setStoreRelation($parentUnzerCredentialsTransfer->getStoreRelationOrFail());` inside

so only 1 return statement will be left